nix/shared.nix

```diff
@@ -189,7 +184,6 @@ let
                     extension
                     doCheckExtension
                     doBenchmarkExtension
-                    doJailbreakExtension
```

Comment:
Oops. I think this caused the `aeson` build to fail due to warnings: https://hydra.dhall-lang.org/build/53056/nixlog/2

Reply:
Nope, that's not it. ;)